examples/run_async_advanced.py

```diff
@@ -28,20 +43,20 @@ async def test(request):
 loop = asyncio.get_event_loop()
 serv_task = asyncio.ensure_future(serv_coro, loop=loop)
 signal(SIGINT, lambda s, f: loop.stop())
-server: AsyncioServer = loop.run_until_complete(serv_task)  # type: ignore
-server.startup()
+server: AsyncioServer = loop.run_until_complete(serv_task)
+loop.run_until_complete(server.startup())
```

Comment:
a) Does this actually work? I recall having trouble getting something like this started (for hosting http and https at the same time on tls docs page).
b) Any chance this could be done without loop arguments?

Reply:
1. Yes, this works. I plan to post the "proper" way to run nested Sanic in the docs later this week.
2. Without the loop? Hmm... I suppose it could wrap it all into a single coroutine. The problem is that `startup` now requires the existence of the loop to be running.